main.py

```diff
@@ -167,6 +167,12 @@ def campaign_group_page(campaign_group=None, pwg=None):
     conn = sqlite3.connect('data.db')
     cursor = conn.cursor()
     sql_args = [campaign_group]
+    counts = cursor.execute("SELECT COUNT(*) FROM samples WHERE campaign_group = ?", sql_args)
+    counts = counts.fetchall()
+    counts = int(counts[0][0])
+    page = int(request.args.get('page', 1))
```

Comment:
I would also wrap this in `max(..., 1)` in case someone decides to put page number -36719 or 0.